src/communication/azext_communication/manual/_help.py

```diff
@@ -407,3 +407,100 @@
         text: |-
                az communication chat message receipt send --thread "19:a-bcd=xyz" --message-id "12345678"
 """
+
+helps['communication rooms'] = """
+    type: group
+    short-summary: Commands to interact with Azure Communication Services Rooms gateway.
+"""
+
+helps['communication rooms create'] = """
+    type: command
+    short-summary: "Creates a new room."
```

Comment:
Please use the first person voice in the help message
```suggestion
    short-summary: "Create a new room."
```

Reply:
Thank you, fixed.